llama-index-integrations/embeddings/llama-index-embeddings-litellm/llama_index/embeddings/litellm/base.py

```diff
@@ -47,6 +49,7 @@ class LiteLLMEmbedding(BaseEmbedding):
             "Only supported in text-embedding-3 and later models."
         ),
     )
+    timeout: Optional[int] = Field(default=60, description="Timeout for each request.", ge=0)
```

Comment:
this adds the parameters to the class, but you aren't actually passing it into `get_embeddings` it looks like?

Reply:
@logan-markewich realy sorry for that, fixed it